plugins/package-managers/node/src/main/kotlin/Yarn.kt

```diff
@@ -110,8 +114,12 @@ class Yarn(
  * Note: The mentioned network issue can be reproduced by setting the network timeout to be very short via the command
  * line option '--network-timeout'.
  */
-internal fun parseYarnInfo(stdout: String): PackageJson? =
-    extractDataNodes(stdout, "inspect").firstOrNull()?.let(::parsePackageJson)
+internal fun parseYarnInfo(stdout: String, stderr: String): PackageJson? =
+    extractDataNodes(stdout, "inspect").firstOrNull()?.let(::parsePackageJson).alsoIfNull {
+        extractDataNodes(stderr, "error").forEach {
+            logger.warn { "Error parsing yarn info: ${it.jsonPrimitive.content}." }
```

Comment:
Nits:
- "yarn" -> "Yarn"
- Probably drop the trailing "." as `content` could already contain it.

Reply:
Also, as these are errors on the Yarn side, should be log to error level, too? This would even make more sense when also parsing for warnings explicitly in a follow-up commit, as these would then be logged as warnings.

Reply:
As ORT's execution still is "successful" (you can rely on the result), I would propose to log the 
errors as "warnings". (And then in a following change log warnings either as info or debug).
Would you agree?

Reply:
Yes, makes sense to me.